gulpfile.js

```diff
@@ -34,17 +34,36 @@ const IMAGES_DIR = 'build/images';
 function parseCommandLine() {
   commander
     .option(
-      '--grep [pattern]',
-      'Run only tests where filename matches a pattern'
+      '--grep <pattern>',
+      'Run only tests where filename matches a regex pattern'
+    )
+    .option('--watch', 'Continuously run tests (default: false)', false)
+    .option('--browser <browser>', 'Run tests in browser of choice.')
+    .option(
+      '--no-browser',
+      "Don't launch default browser. Instead, navigate to http://localhost:9876/ to run the tests."
     )
     .parse(process.argv);
 
-  return {
-    grep: commander.opts().grep,
+  const { grep, watch, browser } = commander.opts();
+  const karmaOptions = {
+    grep: grep,
+    singleRun: !watch,
   };
+
+  // browser option can be either undefined | false | string
+  if (browser === false) {
+    karmaOptions.browsers = null;
+  }
+
+  if (browser) {
```

Comment:
I would suggest to combine this with the above `if` as:

```js
if (browser === false) {
  ...
} else if (browser) {
  ...
}
```

To make it a little more immediately obvious that these are mutually exclusive options.

Reply:
Yes, thanks.